### PR TITLE
most german translation completed

### PR DESCRIPTION
**Pull request by mKorniK**

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -5244,6 +5254,7 @@ function init() {
     ,'Remove %1 selected records?' : {
       cs: 'Odstranit %1 vybraných záznamů'
       ,nb: 'Fjern %1 valgte elementer?'
+      ;de: 'Ausgewählte %1 Aufnahmen löschen?'
       ,ro: 'Șterg %1 înregistrări selectate?'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2470436128/issues/source?bid=2452379&fileBranchId=3259755#l5258)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2470436128/issues/source?bid=2452379&fileBranchId=3259755#l5258)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2470486176/issues/source?bid=2452379&fileBranchId=3259755#l5258)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2470486176/issues/source?bid=2452379&fileBranchId=3259755#l5258)

**Comment by MilosKozak** [comment]
stop adding another PRs until other translations are merged !!!!!!!!!!!!!!
read the guide ....

**Pull request merged**